.github/workflows/ci.yaml

```diff
@@ -12,13 +12,16 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-        - '2.7'
         - '3.0'
         - '3.1'
+        - '3.2'
```

Comment:
can we do 3.3 instead?

Reply:
Thanks, I was supposed to add 3.3, not sure what I did.

3.0 is fine for this PR.  We can drop it whenever we want in the future.  It's a low churn repo with quite fast builds so the only reason to drop 3.0 is it's not easy to keep it here.  I do agree, once we move beyond 3.1 to 3.2/3.3, 3.0 should be dropped.

Reply:
fixed 3.3 missing... tests should be done in a few seconds

Reply:
done!